client/components/jetpack/card/jetpack-rna-dialog-card/index.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import { ReactNode } from 'react';
+import DefaultBackgroundImage from 'calypso/assets/images/jetpack/rna-card-bg.png';
+
+import './style.scss';
+
+interface RnaDialogCardProps {
+	children: ReactNode;
+	cardImage?: string;
+	cardImage2xRetina?: string;
+	isPlaceholder?: boolean;
+}
+
+const JetpackRnaDialogCard: React.FC< RnaDialogCardProps > = ( {
+	children,
+	cardImage,
+	cardImage2xRetina,
+	isPlaceholder,
+} ) => {
+	const cardDesktopSideImage =
+		window.devicePixelRatio > 1 && cardImage2xRetina
+			? cardImage2xRetina
+			: cardImage ?? DefaultBackgroundImage;
+	return (
+		<div
+			className={ classNames( 'jetpack-rna-dialog-card', {
+				'is-placeholder': isPlaceholder,
+			} ) }
+		>
+			<div className="jetpack-rna-dialog-card__body">
+				<div className="jetpack-rna-dialog-card__content">{ children && children }</div>
```

Comment:
I realize this was merged three days ago, but I noticed something while reviewing #73387 and wanted to bring it to y'all's attention:

`{ children && children }` doesn't accomplish anything different than `{ children }`, correct? This seems like a redundant boolean condition we could safely remove.

Reply:
Yes totally, you're right, thank you for spotting this. I'll clean it up on a following PR. 👍